jekyll/_cci2/getting-started.md

```diff
@@ -189,9 +248,9 @@ Read more about workspaces [here](https://circleci.com/docs/2.0/workflows/#using
 ### SSH into Your {% comment %} TODO: Job {% endcomment %}Build
```

Comment:
woah I didn't know it was possible to comment out like that!

Reply:
Haha yeah, I didn't write it, so I learned that recently too :)